### PR TITLE
DDCNLS-5083:add test to check special character in postcode

### PR DESCRIPTION
**Pull request by ashwinirudrappa**

### CODE REVIEW

**Review by Gaur4vGaur — APPROVED** [review]

File: app/forms/BusinessRegistrationForms.scala [hunk 1]
```diff
@@ -38,6 +37,7 @@ object BusinessRegistrationForms {
   val postcodeRegex =
     """(([gG][iI][rR] {0,}0[aA]{2})|((([a-pr-uwyzA-PR-UWYZ][a-hk-yA-HK-Y]?[0-9][0-9]?)|(([a-pr-uwyzA-PR-UWYZ][0-9][a-hjkstuwA-HJKSTUW])|([a-pr-uwyzA-PR-UWYZ][a-hk-yA-HK-Y][0-9][abehmnprv-yABEHMNPRV-Y]))) {0,}[0-9][abd-hjlnp-uw-zABD-HJLNP-UW-Z]{2}))$"""
 
+  val noUkPostCodeRegex = "^[A-Za-z0-9 ]{1,10}$"
```
Comment (Gaur4vGaur) [inline app/forms/BusinessRegistrationForms.scala]: Please follow the naming standards.

File: app/forms/BusinessRegistrationForms.scala [hunk 2]
```diff
@@ -197,10 +198,13 @@ object BusinessRegistrationForms {
       _.isEmpty
     }
     
-    val validatePostCode = ApplicationConfig.validateNonUkClientPostCode(service) && !isAgent
-    if (postCode.isEmpty && validatePostCode) {
+    val postCodeFlag = ApplicationConfig.validateNonUkClientPostCode(service) && !isAgent
```
Comment (Gaur4vGaur) [inline app/forms/BusinessRegistrationForms.scala]: Please name it appropriately.

**Review by Gaur4vGaur — CHANGES_REQUESTED** [review]

**Review by Gaur4vGaur — APPROVED** [review]

File: app/forms/BusinessRegistrationForms.scala (hunk 2 above)
Comment (ashwinirudrappa) [reply app/forms/BusinessRegistrationForms.scala]: reverted back to how it was (validatePostCode)

File: app/forms/BusinessRegistrationForms.scala (hunk 1 above)
Comment (ashwinirudrappa) [reply app/forms/BusinessRegistrationForms.scala]: Fixed it

**Review by Gaur4vGaur — APPROVED** [review]

**Pull request merged**